### ISSUE TITLE
Prompt user experience decredation from 1.39.1 to 1.39.2

### ISSUE THREAD

**Issue by Asguho** [denoland/deno#21812]
When I ran my CLI with the newest Deno version, the user experience was degraded because now the default option isn't editable in the terminal. See the following:
```js
const pageName = prompt("Please edit the name of the page:", "My wrongly spelled page");
console.log(pageName);
```

1.39.1
```bash
PS C:\Users\aske\Documents\GitHub\word-source-cli> deno run .\test.ts
Please edit the name of the page: My wrongly spelled page
```

1.39.2
 ```bash
 PS C:\Users\aske\Documents\GitHub\word-source-cli> deno run .\test.ts
 Please edit the name of the page: [My wrongly spelled page] 
 ```

Besides this, I have also had some issues with not registering "enter" in version 1.39.2 on Windows, but I have not been able to create a reproducible example.

It's super annoying not being able to upgrade to newer versions of Deno and use the latest features because upgrading means making the user experience worse, so if there is anything i can do to help, just let me know.

**Comment by bartlomieju** [comment]
CC @littledivy

**Comment by littledivy** [comment]
This is the same behviour as Deno 1.38.4. We had to revert the PR that allowed editable `prompt()` because of the bugs linked https://github.com/denoland/deno/pull/21739

**Comment by Asguho** [comment]
Will the PR be recommitted?

**Comment by bartlomieju** [comment]
Fixed by https://github.com/denoland/deno/pull/21893, it will be released in Deno v1.39.3 tomorrow.
